Library/Homebrew/extend/os/mac/extend/ENV/super.rb

```diff
@@ -94,6 +94,17 @@ def setup_build_environment(formula = nil)
       self["SDKROOT"] = MacOS.sdk_path
     end
 
+    # Filter out symbols known not to be defined on 10.11 since GNU Autotools
+    # can't reliably figure this out with Xcode 8 on its own yet. The Xcode
+    # version doesn't need to be checked since these are harmless with Xcode 7
+    # even if they're unnecessary.
```

Comment:
I'm not sure about the CLT on 10.11 yet to be honest because Apple releases CLT packages scoped to individual platforms, i.e. there's one available for macOS Sierra but not El Capitan, but when El Capitan gets its version it'll get its own `.pkg`.

It seems feasible they could push a CLT for 10.11 without the problems, but who knows until they do.